app/_hub/critical-blue/approov/index.md

```diff
@@ -3,16 +3,16 @@ name: Approov Mobile App Attestation
 publisher: CriticalBlue Ltd
```

Comment:
The `name:` options was not updated as per the `index.md` file in [this comment](https://github.com/Kong/docs.konghq.com/pull/1951#issuecomment-623436650):

```yml
name: Approov API Threat Protection
publisher: CriticalBlue Ltd
```

Can you please fix this?